docs/source/get-started/package-managers.rst

```diff
@@ -3,20 +3,32 @@ Package Managers
 
 Use your favorite package manager to install Kokkos.
 
-System package managers
+Package managers
```

Comment:
Why are we dropping the distinction between system package managers and other package managers such as Spack and whatnot?

Reply:
I do